packages/react-is/src/ReactIs.js

```diff
@@ -36,6 +37,7 @@ export function typeOf(object: any) {
           case REACT_PROFILER_TYPE:
           case REACT_STRICT_MODE_TYPE:
           case REACT_SUSPENSE_TYPE:
+          case REACT_SUSPENSE_LIST_TYPE:
```

Comment:
Do we still need this change?

Reply:
I think so; without it typeOf will not recognize a SuspenseList and will cause getDisplayNameForReactElement to return "NotImplementedInDevtools" instead of "SuspenseList"